### PR TITLE
Node package and schema tests

### PR DESCRIPTION
**Pull request by NellWaliczek**
This PR replaces #4 and scopes the content down for ease of CR.  It now only does two things: turns the repo into a node package and adds tests to validate the schema.  It does not contain tests to validate the mapping files yet.  That will come in a subsequent PR.
Ignore the changes to README.md; my next step is to fix up that file as part of PR #2 and I'll rebase on top once that's done.

### CODE REVIEW

**Comment by NellWaliczek** [comment]
Fixes #7

**Comment by NellWaliczek** [comment]
Fixes #9

File: package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "gamepad-mappings",
```
Comment (toji) [inline package.json]: Color me surprised that this doesn't already exist, but even still I think something along the lines of "xr-gamepad-mappings" might be more appropriate?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "version": "1.0.0",
```
Comment (toji) [inline package.json]: Is this at a functional-enough state to be considered version `1.0.0`? Don't really care what the version is, honestly, but people tend to think that "1.0" is a statement of stability/release-worthiness.

Comment (NellWaliczek) [reply package.json]: Ah typo. I'll fix it.  I meant to say 0.1.0

File: package.json (hunk 1 above)
Comment (NellWaliczek) [reply package.json]: makes sense.... does that mean the repo name should be xr-gamepad-mappings?

Comment (toji) [reply package.json]: I don't know what npm etiquette is regarding that kind of thing, but probably?

**Pull request merged**